integration_test/third_party_apps_data/applications/elasticsearch/metric_name.txt

```diff
@@ -0,0 +1 @@
+workload.googleapis.com/elasticsearch.node.open_files
```

Comment:
There needs to be a change to the `integration_test/third_party_apps_data/agent/ops-agent/linux/supported_applications.txt` file as well to enable the tests for this application.

Reply:
These tests were already enabled for elasticsearch in #317

Reply:
Ah good point.